Mantle/MTLJSONAdapter.m

```diff
@@ -404,7 +404,12 @@ + (NSDictionary *)valueTransformersForModelClass:(Class)modelClass {
 			if (propertyClass != nil) {
 				transformer = [self transformerForModelPropertiesOfClass:propertyClass];
 			}
-
+			
+			// For User defined MTLModel, we should try to parse it with dictionaryTransformer, since we are inside of json adapter now.
```

Comment:
Can you make this `user-defined`, `JSON` and break the comment at 80 chars? :bow: